cogboard-app/src/main/kotlin/com/cognifide/cogboard/http/HttpClient.kt

```diff
@@ -88,12 +89,9 @@ class HttpClient : AbstractVerticle() {
         val pass = config.getString(CogboardConstants.PROP_PASSWORD) ?: ""
         val token = config.getString(CogboardConstants.PROP_TOKEN) ?: ""
         val headers = config.getJsonObject(CogboardConstants.PROP_HEADERS)
+        val widgetType = config.getString(CogboardConstants.PROP_WIDGET_TYPE) ?: ""
```

Comment:
no widgets here it should be: 
`val authTypes = config.getString(CogboardConstants.PROP_AUTH_TYPES) ?: ""`